inc/core/templates/theme-info-page.php

```diff
@@ -36,23 +36,53 @@
 
 <div class="auth-app-card card">
 <h2>Site Specific Block Patterns</h2>
+<p>Site specific patterns developed specifically for sites can also be inserted from their patterns. Look for  categories that identify the site by name. These patterns can be modified once applied to the page and can either be set up a Reusable Block or alternatively can be copy/pasted between pages or sections once altered.</p>
+
+<?php
+/*
+* CleanBC site specific patterns.
+*/
+if ( defined( 'Bcgov\\Theme\\Block\\CLEANBC\\PATTERNS' ) && CLEANBC\PATTERNS ) :
```

Comment:
still pretty sure the all uppercase is not recommended for class names.

Reply:
Sorry, meant to get to this earlier. Fixed.